mne/bem.py

```diff
@@ -980,7 +980,7 @@ def _check_origin(origin, info, coord_frame='head', disp=False):
 @verbose
 def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
                        volume='T1', atlas=False, gcaatlas=False, preflood=None,
-                       show=False, verbose=None):
+                       show=False, verbose=None, force_copy=False):
```

Comment:
rather than making the API more complex you should try to detect the pb and switch to a copy with a warning message.